release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -287,6 +287,19 @@ With this release, {cgu-operator-first} uses a {rh-rhacm-first} feature to remed
 
 For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.adoc#talo-policies-concept_cnf-topology-aware-lifecycle-manager[Update policies on managed clusters].
 
+[id="ocp-4-16-edge-computing-accelerated-ztp_{context}"]
+==== Accelerated provisioning of {ztp} (Technology Preview)
+
+With this release, you can reduce the time taken for cluster installation by using accelerated provisioning of {ztp} for {sno}. 
+Accelerated ZTP speeds up installation by applying Day 2 manifests derived from policies at an earlier stage.
+
+The benefits of accelerated provisioning of {ztp} increase with the scale of your deployment. 
+Full acceleration gives more benefit on a large number of clusters. 
```

Comment:
Should this be `larger number of clusters`?

Reply:
thanks for the review  @JoeAldinger 
Sure, "larger" it is.